e2e/README.md

```diff
@@ -5,7 +5,6 @@ The package contains Dial end-to-end tests. Tests are implemented using [Playwri
 ## Prerequisites
 
 Tests are using Auth0 credentials to access an application.
-Please, set `AUTH_TEST_TOKEN` variable in .env file in order to enable it.
```

Comment:
instead of removing, need to add: Please, set `E2E_USERNAME` and `E2E_PASSWORD` variables in .env file in order to perform login operation.